controllers/article-controllers.js

```diff
@@ -0,0 +1,19 @@
+const { fetchArticle } = require("../models/articles-models");
+
+exports.getArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticle(article_id)
+    .then((article) => {
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
```

Comment:
I would remove this patch request function from this branch and keep each branch as a method and endpoint, as opposed to a branch for an endpoint and all its associated methods.